src/api/routes/auth/register.ts

```diff
@@ -225,6 +225,14 @@ router.post(
 		}
 
 		if (body.password) {
+                        if(body.password.length<8){
```

Comment:
This looks like it would clash with 

register_password_minLength
register_password_minNumbers
register_password_minSymbols

in the database config.
I am pretty sure your implementation does not account for this